assets/generated/styles.bundle.css

```diff
@@ -0,0 +1,5030 @@
+html {
```

Comment:
Generated budnles should not be checked into git. They should be build as needed in development. In production they will be built by the node-js heroku buildpack and then the python buidpack will run django collectstatic.